kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/ErrantRecordHandler.java

```diff
@@ -33,6 +34,18 @@ public void sendRecordsToDLQ(Set<SinkRecord> rows, Exception e) {
         }
     }
 
+    public void sendRecordsToDLQ(Map<SinkRecord, Throwable> rowToError) {
+        if(errantRecordReporter != null) {
+            logger.debug("Sending {} records to DLQ", rowToError.size());
+            for (Map.Entry<SinkRecord, Throwable> rowToErrorEntry : rowToError.entrySet()) {
+                // Reporting records in async mode
+                errantRecordReporter.report(rowToErrorEntry.getKey(), rowToErrorEntry.getValue());
```

Comment:
is there any error handling in case sending to the reporter fails? How is it done for other connectors?

Reply:
No the task would fail in this case. 
@aniketshrimal are you aware which other connector have DLQ implemented and how is failure handled ?